googletest/googlemock/msvc/2015/gmock.vcxproj

```diff
@@ -103,7 +103,7 @@
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <ClCompile>
       <Optimization>Disabled</Optimization>
-      <AdditionalIncludeDirectories>..\..\include;..\..;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>

```

Comment:
>AdditionalIncludeDirectories [](start = 7, length = 28)

Just curious, could we use cmake to generate the project file form googletest repo under third_party, instead of check in the project files directly?

Reply:
Good point. I tried that but the googletest build fails for win32. The problem is cmake by default generates solution files either for 32 bit or 64 bit based on generator options. The custom solution files include both. Also, there are a few changes in those files over time to speed up the build and didn't want to lose them.
We can probably look at it separately if it's possible to get rid of it?

Reply:
Consider it in future PR is fine.